src/sc/sidechain.h

```diff
@@ -103,6 +104,14 @@ class CSidechain {
     // value = the immature amount
     std::map<int, CAmount> mImmatureAmounts;
 
+    // the last ftScFee and mbtrScFee values, as set by the active certificates
+    std::list<Sidechain::ScFeeData> scFees;
+
+    // memory only
+    int sizeOfScFeesContainers;
```

Comment:
As I understood it is the `maxSize...`. Let's rename

Reply:
Done